### PR TITLE
Add some basic capsule tests

### PR DESCRIPTION
**Pull request by stbenjam**
For use with https://github.com/Katello/forklift/pull/248

### CODE REVIEW

**Comment by ehelms** [comment]
Do you want to add this to the bats role? Right now it runs only content tests (https://github.com/Katello/forklift/blob/master/playbooks/roles/bats/tasks/main.yml#L18). I had to run it like this because nothing I did could get the more normal 'katello-bats content' style command to work :/

File: playbooks/roles/bats/tasks/main.yml [hunk 1]
```diff
@@ -9,12 +9,9 @@
...
-- name: 'Copy bats'
-  shell: >
-    mv /bats/libexec/* /usr/bin/
-
 - name: 'Run bats'
-  shell: >
-    /root/forklift/bats/fb-content-katello.bats
+  shell: /usr/local/bin/katello-bats content capsule
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/usr/local/bin"
```
Comment (stbenjam) [inline playbooks/roles/bats/tasks/main.yml]: This seems to fix katello-bats running for me

Comment (ehelms) [inline playbooks/roles/bats/tasks/main.yml]: Is the `chdir` on lines 16/17 still needed?

**Comment by stbenjam** [comment]
Tested w/out chdir and seems to work for me, updated

**Comment by ehelms** [comment]
ACK! @stbenjam awesome!

**Pull request merged**